staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy.go

```diff
@@ -98,6 +105,9 @@ func proxyError(w http.ResponseWriter, req *http.Request, error string, code int
 }
 
 func (r *proxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+	ctx, span := tracing.Start(req.Context(), aggregatorComponent)
```

Comment:
I don't think this span is needed.  You should already get a span from the DefaultHandlerChain.

Reply:
i added this mostly to identify that the request is running through the aggregator's proxy. i can remove it